pages/pipelines/create_your_own.md

```diff
@@ -66,6 +66,7 @@ To create a new pipeline:
  _New Pipeline_.
 
     If you're prompted to connect your repositories, we recommend completing that first, but you can always connect them later from the settings.
+    After you have connected your repositories, you'll be able to select them from the dropdown, and enable automatic webhook creation.
```

Comment:
Nitpicks: Reducing some "you" and mentioning where the dropdown is.

```suggestion
    After connecting your repositories, you can select them from the dropdown during pipeline creation and enable automatic webhook creation.
```

Reply:
Love the suggestion, added ✅